### PR TITLE
Abnormal Security Connector

### PR DESCRIPTION
**Pull request by cs-omkar-kumbhar**
PMDB: 27389

#### UTCs:
- [x] Connector installation verified.
- [x] Connector logo verified.
- [x] Actions and Playbook list verified.
- [x] Playbook tags verified.
- [x] All Playbook are in info mode verified.
- [x] All Playbook are in inactive mode verified.

### CODE REVIEW

File: abnormal-security/info.json [hunk 1]
```diff
@@ -0,0 +1,1259 @@
...
+  "icon_large_name": "large.png",
+  "help_online": "",
+  "configuration": {
+    "fields": [
+      {
+        "title": "Server URL",
+        "name": "server_url",
+        "required": true,
+        "editable": true,
+        "visible": true,
```
Comment (cs-adarsh-gupta) [inline abnormal-security/info.json:20]: Tooltips are missing in Configuration fields.

File: abnormal-security/info.json [hunk 2]
```diff
@@ -0,0 +1,1259 @@
...
+        "type": "text",
+        "description": "Specify the URL of the Abnormal Security server to connect and perform the automated operations."
+      },
+      {
+        "title": "Access Token",
+        "name": "access_token",
+        "required": true,
+        "editable": true,
+        "visible": true,
```
Comment (cs-adarsh-gupta) [inline abnormal-security/info.json:29]: Tooltip missing

File: abnormal-security/info.json [hunk 3]
```diff
@@ -0,0 +1,1259 @@
...
+        "type": "password",
+        "description": "Specify the access token used to access the Abnormal Security server to connect and perform the automated operations."
+      },
+      {
+        "title": "Verify SSL",
+        "description": "Specifies whether the SSL certificate for the server is to be verified. By default, this option is set to True.",
+        "required": false,
+        "editable": true,
+        "visible": true,
```
Comment (cs-adarsh-gupta) [inline abnormal-security/info.json:38]: Tooltip missing

File: abnormal-security/info.json [hunk 4]
```diff
@@ -0,0 +1,1259 @@
...
+          "visible": true,
+          "editable": true,
+          "type": "checkbox",
+          "name": "mock-data",
+          "tooltip": "Returns test data if set to True.",
+          "description": "Returns test data if set to True.",
+          "value": false
+        }
+      ],
+      "output_schema": {}
```
Comment (cs-adarsh-gupta) [inline abnormal-security/info.json]: Output Schema is missing here, CSV IRI info should be given as output.

File: abnormal-security/info.json [hunk 5]
```diff
@@ -0,0 +1,1259 @@
...
```
Comment (cs-adarsh-gupta) [inline abnormal-security/info.json]: If the output of action is in csv format, Shouldn't we upload it to FSR Attachment module and output it's IRI info?

File: abnormal-security/info.json [hunk 6]
```diff
@@ -0,0 +1,1259 @@
...
```
Comment (cs-adarsh-gupta) [inline abnormal-security/info.json]: If this action has no output and just returns a status code, Then if the status code is 200 We can return Submitted Successfully type of output

File: abnormal-security/info.json [hunk 7]
```diff
@@ -0,0 +1,1259 @@
...
+          "title": "Inquiry Type",
+          "required": true,
+          "visible": true,
+          "editable": true,
+          "type": "select",
+          "name": "inquiry_type",
+          "tooltip": "Specify whether to fetch missed attacks or false positives.",
+          "description": "Specify whether to fetch missed attacks or false positives.",
+          "options": [
+            "MISSED_ATTACK",
```
Comment (cs-adarsh-gupta) [inline abnormal-security/info.json]: Make the options more friendly looking for UI.

File: abnormal-security/info.json [hunk 8]
```diff
@@ -0,0 +1,1259 @@
...
+          "title": "Status",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "multiselect",
+          "name": "status",
+          "tooltip": "Only Detection 360° cases with these statuses will be retrieved. Defaults to all statuses.",
+          "description": "Only Detection 360° cases with these statuses will be retrieved. Defaults to all statuses.",
+          "options": [
+            "UNREVIEWED",
```
Comment (cs-adarsh-gupta) [inline abnormal-security/info.json]: Make the options more friendly looking for UI.

File: abnormal-security/info.json [hunk 9]
```diff
@@ -0,0 +1,1259 @@
...
+          "title": "Source",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "select",
+          "name": "source",
+          "tooltip": "Filters threats based on the source of detection. Default value is all.",
+          "description": "Filters threats based on the source of detection. Default value is all.",
+          "options": [
+            "all",
```
Comment (cs-adarsh-gupta) [inline abnormal-security/info.json]: Make the options more friendly looking for UI.

File: abnormal-security/info.json (hunk 1 above)
Comment (cs-omkar-kumbhar) [reply abnormal-security/info.json:20]: not required

File: abnormal-security/info.json (hunk 2 above)
Comment (cs-omkar-kumbhar) [reply abnormal-security/info.json:29]: not required

File: abnormal-security/info.json (hunk 3 above)
Comment (cs-omkar-kumbhar) [reply abnormal-security/info.json:38]: not required

File: abnormal-security/info.json (hunk 4 above)
Comment (cs-omkar-kumbhar) [reply abnormal-security/info.json]: done

File: abnormal-security/info.json (hunk 5 above)
Comment (cs-omkar-kumbhar) [reply abnormal-security/info.json]: done

File: abnormal-security/info.json (hunk 6 above)
Comment (cs-omkar-kumbhar) [reply abnormal-security/info.json]: done

File: abnormal-security/info.json (hunk 7 above)
Comment (cs-omkar-kumbhar) [reply abnormal-security/info.json]: done

File: abnormal-security/info.json (hunk 8 above)
Comment (cs-omkar-kumbhar) [reply abnormal-security/info.json]: done

File: abnormal-security/info.json (hunk 9 above)
Comment (cs-omkar-kumbhar) [reply abnormal-security/info.json]: done

**Review by cs-adarsh-gupta — APPROVED** [review]

**Pull request merged**